web/react/components/rhs_comment.jsx

```diff
@@ -12,12 +12,16 @@ var FileAttachmentList = require('./file_attachment_list.jsx');
 var client = require('../utils/client.jsx');
 var AsyncClient = require('../utils/async_client.jsx');
 var ActionTypes = Constants.ActionTypes;
+var twemoji = require('twemoji');
 
 export default class RhsComment extends React.Component {
     constructor(props) {
         super(props);
 
         this.retryComment = this.retryComment.bind(this);
+        this.parseEmojis = this.parseEmojis.bind(this);
+        this.componentDidMount = this.componentDidMount.bind(this);
+        this.componentDidUpdate = this.componentDidUpdate.bind(this);
```

Comment:
For native React functions you don't need to bind this. So componentDidMount and componentDidUpdate are unnecessary here.